### ISSUE TITLE
What is “map-id” for?

### ISSUE THREAD

**Issue by alreadytiredofnomcom** [alreadytiredofnomcom/L2SMissues#33]
Bin:
What is the “map-id” here for?

```
 rw ethernet-connection
|           |  +--rw vlan
|           |  |  +--rw svlan-id-ethernet-tag?     string
|           |  |  +--rw cvlan-id-to-evc-map* [map-id]
|           |  |     +--rw map-id         string
```

Qin:
Not sure the mapping relation from clan-id to evc, is 1:1 or 1:n If it is 1 to n, cvlan-id-to-evc-map should be defined as list or leaflist, if map-id is not needed, we can define cvlan-id-to-evc-map as leaflist.
If it is 1 to 1, cvlan-id-to-evc-map should be defined as string type or as container.

**Comment by alreadytiredofnomcom** [comment]
Only need it under the service and once under UNI/ENNI

**Comment by alreadytiredofnomcom** [comment]
Available on 20160926 release
